### PR TITLE
AVRO-2815: Remove the ant stuff

### PR DESCRIPTION
**Pull request by Fokko**
Make sure you have checked _all_ steps below.

### Jira

- [ ] My PR addresses the following [Avro Jira](https://issues.apache.org/jira/browse/AVRO/) issues and references them in the PR title. For example, "AVRO-1234: My Avro PR"
  - https://issues.apache.org/jira/browse/AVRO-XXX
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).

### Tests

- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [ ] My commits all reference Jira issues in their subject lines. In addition, my commits follow the guidelines from "[How to write a good git commit message](https://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain Javadoc that explain what it does

### CODE REVIEW

File: doc/src/content/xdocs/gettingstartedpython.xml [hunk 1]
```diff
@@ -73,9 +80,8 @@ $ python
       </p>
       <source>
 $ cd lang/py/
-$ python setup.py install
+$ pip install -e .
```
Comment (kojiromike) [inline doc/src/content/xdocs/gettingstartedpython.xml:83]: I suggest we write this `python3 -m pip install -e .`

Comment (Fokko) [reply doc/src/content/xdocs/gettingstartedpython.xml]: Good point, thanks

File: doc/src/content/xdocs/gettingstartedpython.xml [hunk 2]
```diff
@@ -51,8 +51,15 @@
     <section id="download_install">
       <title>Download</title>
       <p>
-        Avro implementations for C, C++, C#, Java, PHP, Python, and Ruby can be
-        downloaded from the <a
+        For Python, the easiest way to get started is to install it from PyPI.
+        Python's Avro API is available over <a href="https://pypi.org/project/avro/">PyPi</a>.
+      </p>
+      <source>
+$ pip install avro
```
Comment (kojiromike) [inline doc/src/content/xdocs/gettingstartedpython.xml:58]: Ah, shoot, sorry I missed this, we should probably do it the same as below with `python3 -m pip`.

Comment (Fokko) [reply doc/src/content/xdocs/gettingstartedpython.xml]: No worries, I've updated this one as well.

**Comment by Fokko** [comment]
Thanks for the review @kojiromike @RyanSkraba

**Pull request merged**